index.html

```diff
@@ -49,5 +49,167 @@ <h2>I'am a Sotfware Developer</h2>
           </div>
         </section>
     </main>
+    <div class="main-body">  
+      <section class="main-row">
+        <h1>My Recent work</h1>
```

Comment:
- [ ] Let's not use more that one `h1` tag in the entire web app. This will affect the accesibility and SEO of your page.[ You can read more on the subject here.](https://www.silverdisc.co.uk/blog/2020/11/06/how-important-are-h1-tags-seo). Instead, use the other tags for titles: `h2...h6`.